### PR TITLE
OSDOCS-5178:Allow users to create more than one sc

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OSDOCS-5178

Link to docs preview:https://55822--docspreview.netlify.app/openshift-enterprise/latest/storage/dynamic-provisioning.html#change-default-storage-class_dynamic-provisioning

QE review:
- [ ] QE has approved this change.

Additional information:

**PTAL**: @jsafrane, @ropatil010, @gcharot

### CODE REVIEW

File: modules/dynamic-provisioning-change-default-class.adoc [hunk 1]
```diff
@@ -25,20 +34,26 @@ NAME                 TYPE
...
 ----
++
+[NOTE]
+====
+It is acceptable to have multiple default storage classes for a short time. However, you should ensure that eventually only one default storage class exists.
+// add xref to multi/no default SC module
+====
 
-. Make another storage class the default by setting the `storageclass.kubernetes.io/is-default-class` annotation to `true`:
+. Change the value of the `storageclass.kubernetes.io/is-default-class` annotation to `false` for the previous default storage class:
```
Comment (aldiazRH) [inline modules/dynamic-provisioning-change-default-class.adoc:52]:
```suggestion
. Change the value of the `storageclass.kubernetes.io/is-default-class` annotation to `false` by running the following command:
```

**Review by aldiazRH — COMMENTED** [review]
Looks good, just a few suggestions here and there. Another reviewer might take a look at it as well. :+1:

**Comment by jeana-redhat** [comment]
Checking behind for peer review onboarding, @aldiazRH's comments look right to me.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by lpettyjo** [comment]
@lpettyjo be sure to add xref to multi/no default sc module (line 49).

File: modules/dynamic-provisioning-change-default-class.adoc [hunk 2]
```diff
@@ -25,20 +34,26 @@ NAME                 TYPE
...
 +
 [source,terminal]
 ----
-$ oc patch storageclass gp2 -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "false"}}}'
+$ oc patch storageclass standard -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'
 ----
++
+[NOTE]
+====
+It is acceptable to have multiple default storage classes for a short time. However, you should ensure that eventually only one default storage class exists.
```
[the remark this answers is not in the document]
Comment (gcharot) [reply modules/dynamic-provisioning-change-default-class.adoc:48]: I would highlight what happens in that case (the most recently created SC is used)

File: modules/dynamic-provisioning-change-default-class.adoc [hunk 3]
```diff
@@ -27,20 +34,26 @@ NAME                 TYPE
...
 +
 [source,terminal]
 ----
-$ oc patch storageclass gp3 -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "false"}}}'
+$ oc patch storageclass standard -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'
 ----
++
+[NOTE]
+====
+It is acceptable to have multiple default storage classes for a short time. However, you should ensure that eventually only one default storage class exists.
```
Comment (gcharot) [inline modules/dynamic-provisioning-change-default-class.adoc:48]: I would highlight what happens in that case (the most recently created SC is used)

Comment (ropatil010) [reply modules/dynamic-provisioning-change-default-class.adoc:48]: Yeah, according to latest sc time stamp.

Comment (lpettyjo) [reply modules/dynamic-provisioning-change-default-class.adoc:48]: @lpettyjo remember to add xref to multi/no sc default sc module in note.

File: modules/dynamic-provisioning-change-default-class.adoc [hunk 4]
```diff
@@ -27,20 +34,26 @@ NAME                 TYPE
...
+// add xref to multi/no default SC module
+====
 
-. Make another storage class the default by setting the `storageclass.kubernetes.io/is-default-class` annotation to `true`:
+. Change the value of the `storageclass.kubernetes.io/is-default-class` annotation to `false` for the previous default storage class:
 +
 [source,terminal]
 ----
-$ oc patch storageclass standard -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'
+$ oc patch storageclass gp2 -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "false"}}}'
```
[the remark this answers is not in the document]
Comment (lpettyjo) [reply modules/dynamic-provisioning-change-default-class.adoc:56]: Good catch. Thanks!

File: modules/dynamic-provisioning-change-default-class.adoc (hunk 3 above)
Comment (lpettyjo) [reply modules/dynamic-provisioning-change-default-class.adoc]: Done.

**Comment by ropatil010** [comment]
LGTM
Thanks!

**Comment by jsafrane** [comment]
lgtm

**Comment by gcharot** [comment]
lgtm

**Pull request merged**